src/hardware/gameblaster.cpp

```diff
@@ -222,6 +228,8 @@ void GameBlaster::AudioCallback(const int requested_frames)
 {
 	assert(channel);
 
+	std::lock_guard lock(mutex);
```

Comment:
General question not related to just the Game Blaster. Have you done measurements of how much blocking (if any) happens due to these locks? My assumption is that waiting would happen very infrequently; is that true?

Reply:
I don't think it should be too bad but I'm not sure how to measure lock contention.

Reply:
Yeah I'm not sure either 😄